lib/Serialization/Deserialization.cpp

```diff
@@ -5799,6 +5799,9 @@ class SwiftToClangBasicReader :
 
 llvm::Expected<const clang::Type *>
 ModuleFile::getClangType(ClangTypeID TID) {
+  if (getContext().LangOpts.UseClangFunctionTypes)
+    return nullptr;
```

Comment:
Are you sure about the direction of this check? Or if we skip `getClangType` when `UseClangFunctionTypes`, what do we use instead?

Reply:
My bad, I just realized this when running tests. It's incorrect.

Reply:
Fixed polarity of check.